tests/features/bootstrap/FeatureContext.php

```diff
@@ -29,6 +35,229 @@ public function assertAccessDenied() {
   }
 
   /**
+   * Assert that certain fields are present on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
+   *   Thrown when an expected field is not present.
+   *
+   * @Then /^(?:|the following )fields should be present? "(?P<fields>[^"]*)"$/
+   */
+  public function assertFieldsPresent($fields) {
+    $fields = $this->explodeCommaSeparatedStepArgument($fields);
+    $not_found = [];
+    foreach ($fields as $field) {
+      $is_found = $this->getSession()->getPage()->find('named', ['field', $field]);
```

Comment:
The whole chain to the `DocumentElement` is just returning properties, so it's a micro-optimisation, but it's ok. Good one for the `findField()` method.